sdk/communication/Azure.Communication.JobRouter/README.md

```diff
@@ -157,10 +157,10 @@ foreach (EventGridEvent egEvent in egEvents)
     switch (egEvent.EventType)
     {
         case "Microsoft.Communication.WorkerOfferIssued":
-            AcsRouterWorkerOfferIssuedEventData deserializedEventData =
+            AcsRouterWorkerOfferIssuedEventData? deserializedEventData =
```

Comment:
Weird that they have nullables turned on for this package.  I thought we did that only in Core.

Reply:
We should enable it for all SCM-based clients! 😁